### ISSUE TITLE
use_before_filter numeric value

### ISSUE THREAD

**Issue by michaelachrisco** [ndea/regressor#12]
After running `rails generate regressor:controller`, some of the controllers generated a numeric value where there were none. These were only with inherited controllers such as:
### Controller:

``` ruby
class User::RolesController < RolesController; end
```
### Spec:

``` ruby
describe User::RolesController do
    it { should use_before_filter(:70303058551480) }
```

**Comment by erwinjunker** [comment]
I experienced this behaviour before - using cancan or cancancan. Do you use any of them?

**Comment by michaelachrisco** [comment]
Pundit actually.

**Comment by erwinjunker** [comment]
Should be fixed in version 0.5.7. It was actually a problem using a proc for a before filter.
